infrastructure/src/main/resources/reference.conf

```diff
@@ -1,11 +1,9 @@
 influx {
-  url = "https://scala-ci.typesafe.com/influx/"
-  user = "scala"
+  url = "http://localhost:8086"
```

Comment:
Will this be available from the outside? Shouldn't we use here a URL that Fabien gives us?

Reply:
Not yet. There are InfluxDB and Grafana instances on our server, but it is not open from outside EPFL. Let's see what we'll do...